cmd/manager/server.go

```diff
@@ -42,11 +36,8 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsHost              = "0.0.0.0"
-	metricsPort        int32 = 8383
-	metricsServicePort       = []v1.ServicePort{
-		{Port: metricsPort, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metricsPort}},
-	}
+	metricsHost       = "0.0.0.0"
```

Comment:
s/var/const 

Also might be worth updating the comment to say the k8s service needs to be updated if this is changed.

Reply:
:+1: done